nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters
# Lw add edit view categories2

# Description
Added the add, edit, and view feature for Categories, also all categories are now listed alphabetically.

# Ticket Reference

> [#9 - View All Categories](https://github.com/nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters/issues/9)
> [#10 - Create a Category](https://github.com/nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters/issues/10)
> [#12 - Edit Categories](https://github.com/nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters/issues/12)


## Type of change


# Checklist:

# Testing Instructions:

Run the API in Visual studio, and npm serve the client folder.
Type localhost:3000/users into the browser bar.
Visually confirm that you're rerouted back to the login page.
Start program with ISS Express
Go to localhost:5001 in your browser.
-Login with "admin@example.com"
Navigate to the Categories link in the navbar.
Click Categories in the NavBar to go back to the list of posts
Click the New Category button to navigate to the create new post form
Complete all input fields and click the Submit button
Clicking that button will create a new post a nd navigate you to the detail view for it.
